test/cqlpy/util.py

```diff
@@ -108,13 +108,27 @@ def keyspace_has_tablets(cql, keyspace):
 #   with new_test_keyspace(cql, '...') as keyspace:
 # This is not a fixture - see those in conftest.py.
 @contextmanager
-def new_test_keyspace(cql, opts):
-    keyspace = unique_name()
-    cql.execute("CREATE KEYSPACE " + keyspace + " " + opts)
+def new_test_keyspace(cql, opts, name=unique_name()):
+    cql.execute("CREATE KEYSPACE " + name + " " + opts)
     try:
-        yield keyspace
+        yield name
     finally:
-        cql.execute("DROP KEYSPACE " + keyspace)
+        cql.execute("DROP KEYSPACE " + name)
+
+# A utility function for creating a new temporary table with a given schema and name.
+# It can be used in a "with", as:
+#   with new_named_test_table('...') as table:
+# This is useful when you need a table with a specific name for your test.
+# If a specific table name is not required in the test case,
+# prefer using new_test_table() to avoid name collisions.
+@contextmanager
+def new_named_test_table(cql, ks_name, tbl_name, schema, extra=""):
+    qualified_table_name = ks_name + "." + tbl_name
+    cql.execute("CREATE TABLE " + qualified_table_name + "(" + schema + ")" + extra)
+    try:
+        yield qualified_table_name
+    finally:
+        cql.execute("DROP TABLE " + qualified_table_name)
```

Comment:
Again, please don't move new things to util.py just because you want to use them in one file.
new_named_test_table() can remain in the single file that uses it, until we're sure we need it from multiple files.

I'm not against new_test_keyspace taking a name parameter. I think it's a worthy addition.  But note we have a lot of tests (e.g, in test_keyspace.py) that could have used such a feature and don't.